be/src/exec/pipeline/driver_limiter.h

```diff
@@ -0,0 +1,46 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+
+#include <atomic>
+#include <memory>
+
+#include "common/statusor.h"
+#include "gutil/strings/substitute.h"
+
+namespace starrocks::pipeline {
+
+class DriverLimiter {
```

Comment:
Is this class necessary?

Reply:
@kangkaisen Otherwise, this logic will be put to ExecEnv, which contains too many logics.
And by abstracting this logic to a class, maybe we could replace the reject strategy more easily in the future.